### ISSUE TITLE
Nohboard is freezing when I tab over in-game

### ISSUE THREAD

**Issue by yossa-b** [ThoNohT/NohBoard#160]
Hello, I'm new and just made an acct to post this since I can't seem to find another solution anywhere. I've tried running nohboard as administrator in windows 10, as well as running the game itself in borderless fullscreen. I've installed and reinstalled it a few times, i'm not really sure what else to try. I have two monitors, and I've tried dragging nohboard to the second monitor, as some suggested, but it still stopped when I tabbed over to the game. I have nohboard running with the intention of capturing my mouse movement only, it's the simple circle and radial movement indicator. I have this going with input overlay in OBS. I wonder if there is something else in OBS I could do? Anyway, thanks for the help regardless, it is an amazing tool, I just want to get it to work, lol.

**Comment by deific-jam** [comment]
try running nohboard as administrator

**Comment by yossa-b** [comment]
Yep, I've tried: running noh and obs as administrator, running the game in borderless fullscreen, used both dual and single monitor set up and none of it worked. Perhaps it's a windows or obs setting?

**Comment by yossa-b** [comment]
or a mouse setting, i use a logitech g900.

**Comment by yossa-b** [comment]
It's not so much a "freeze," but more like nohboard stops capturing my mouse input once I'm in game. I've tried this with Doom Eternal, maybe I should try other games as well.

**Comment by yossa-b** [comment]
https://user-images.githubusercontent.com/85777722/121795056-e8cfef00-cbc1-11eb-9619-eae3dc7c5be4.mp4

**Comment by yossa-b** [comment]
It's weird, the flickers like that once in-game like there is some amount of input capture still happening, but it's not as full or complete as when I'm just in my desktop.

**Comment by argash** [comment]
Not sure if it's related or not so I'll mention here first and make a new issue if requested. When I ALT+TAB it will lock those two keys as on until I hit them again to turn them off.

**Comment by Tpurokz** [comment]
Ya same here bro but mine on Delta force, didnt know what to do? If anyone here could please explain what am i doin wrong in OBS?
Thanks btw. :)

**Comment by MacHawkDown1** [comment]
> Ya same here bro but mine on Delta force, didnt know what to do? If anyone here could please explain what am i doin wrong in OBS? Thanks btw. :)

I was having the same issue with rivals.

Close Noh
Close OBS/Slobs
Run OBS/Slobs as admin
Run NoH as admin
Remove and re-add Noh
